src/org/wso2/tg/jenkins/Properties.groovy

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.wso2.tg.jenkins
+
+@Singleton
+class Properties {
+
+    static def TESTGRID_NAME                = "WSO2-TestGrid"
+    static def TESTGRID_DIST_LOCATION       = "/testgrid/testgrid-home/testgrid-dist/"
+    static def TESTGRID_HOME                = "/testgrid/testgrid-home/"
+    static def JOB_CONFIG_YAML              = "job-config.yaml"
+    static def SQL_DRIVERS_LOCATION_UNIX    ="/home/centos/sql-drivers/"
+    static def SQL_DRIVERS_LOCATION_WINDOWS ="/testgrid/sql-drivers"
+    static def REMOTE_WORKSPACE_DIR_UNIX    ="/opt/wso2/workspace"
+    static def REMOTE_WORKSPACE_DIR_WINDOWS ="c:/testgrid/workspace"
```

Comment:
Groovy can handle the path separator differences?

Reply:
This is not used with in groovy rather it is used in the python scripts.